web/packages/teleterm/src/ui/DocumentCluster/DocumentCluster.story.tsx

```diff
@@ -220,6 +221,28 @@ export const OnlineLoadedResources = () => {
           },
           { kind: 'database', resource: makeDatabase() },
           { kind: 'kube', resource: makeKube() },
+          { kind: 'app', resource: { ...makeApp(), name: 'TCP app' } },
+          {
+            kind: 'app',
+            resource: {
+              ...makeApp(),
+              name: 'HTTP app',
+              endpointUri: 'http://localhost:8080',
+            },
+          },
+          {
+            kind: 'app',
+            resource: {
+              ...makeApp(),
+              name: 'AWS console',
+              endpointUri: 'cloud://localhost:8080',
```

Comment:
From what I see (in teleport-14-ent), aws console apps have endpoints starting with https.